pkg/kv/kvnemesis/validator.go

```diff
@@ -532,6 +533,56 @@ func (v *validator) processOp(op Operation) {
 		if v.buffering == bufferingSingle {
 			v.checkAtomic(`deleteRangeUsingTombstone`, t.Result)
 		}
+	case *AddSSTableOperation:
+		if resultIsErrorStr(t.Result, "key range .* outside of bounds of range") {
+			// The AddSSTable may race with a range split. It's not possible to ingest
+			// an SST spanning multiple ranges, but the generator will optimistically
+			// try to fit the SST inside one of the current ranges, so we ignore the
+			// error and try again later.
+		} else if resultIsErrorStr(t.Result, "write for key .* at timestamp .* too old; wrote at") {
+			// We may race with a concurrent write. Try again later.
+		} else if v.checkNonAmbError(op, t.Result) {
```

Comment:
I definitely am not doing anyone a favor with this `v.checkX` methods, I needed to read comments two layers deep to remember how this works again. When looking at this line I was like "are you just ignoring errors here?!" but `v.checkNonAmbError` will emit a failure to the validator on non-exempt errors.

Reply:
Added a comment to clarify this.